src/Umbraco.Core/Persistence/Repositories/Implement/MemberGroupRepository.cs

```diff
@@ -115,6 +115,16 @@ protected override void PersistUpdatedItem(IMemberGroup entity)
             entity.ResetDirtyProperties();
         }
 
+        public IMemberGroup Get(Guid uniqueId)
+        {
+            var sql = GetBaseQuery(false);
+            sql.Where("umbracoNode.uniqueId = @uniqueId", new { uniqueId });
```

Comment:
This could be strongly typed with `Where<NodeDto>` like we do in lots of places

Reply:
@Shazwazza yes it could, but I think there is several other places where the SQL-queries can be re-written with strongly typed syntax. Currently we have many of these or combination of strongly typed query combined with string SQL-queries where it could use `.Where<SomeDto>` or `.WhereIn<SomeDto>`.

Could be enhanced further in some other PR's to clean this up 👍 

At least I replaced the "umbracoNode" table with the constant value 😁

Reply:
yep, just wanted to mention it :)

Reply:
@Shazwazza I have updated this as part of https://github.com/umbraco/Umbraco-CMS/pull/9467 🎄